tests/API/Management/LogsTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Auth0\Tests\API;
+
+use Auth0\SDK\API\Management;
+
+/**
+ * Class LogsTest.
+ * Tests the Auth0\SDK\API\Management\Logs class.
+ *
+ * @package Auth0\Tests\API
+ */
+class LogsTest extends ApiTests
+{
+
+    /**
+     * Logs API client.
+     *
+     * @var mixed
+     */
+    protected static $api;
+
+    /**
+     * Valid Log ID to test getter.
+     *
+     * @var string
+     */
+    protected static $log_id;
+
+    /**
+     * Sets up API client for entire testing class.
+     *
+     * @return void
+     */
+    public static function setUpBeforeClass()
+    {
+        $env   = self::getEnvStatic();
+        $token = self::getTokenStatic($env, [ 'logs' => [ 'actions' => ['read'] ] ]);
+        $api   = new Management($token, $env['DOMAIN']);
+
+        self::$api = $api->logs;
+    }
+
+    /**
+     * Test a general search.
+     *
+     * @return void
+     */
+    public function testLogSearch()
+    {
+        $search_results = self::$api->search();
+        $this->assertNotEmpty($search_results);
+
+        // Get the 10th log entry to test pagination in self::testLogSearchPagination().
```

Comment:
I don't see this being used in the test `testLogSearchPagination` as this comment mentions. Anyway, it shouldn't depend on it. Please write tests that are independent from each other.